data/transactions/logic/assembler.go

```diff
@@ -860,6 +868,14 @@ func simpleImm(value string, label string) (byte, error) {
 	return byte(res), err
 }
 
+func int8Imm(value string, label string) (byte, error) {
```

Comment:
minor question about this function: is this one related with frame pointer, or it seems just used in itxn field?

Reply:
nvm, seems frame_dig/frame_bury are using them tho? is that correct?

Reply:
Yes, they are the only instructions that use a signed immediate.  They do that so they can look _below_ the fp for arguments, and _above_ for placing the return value(s) and local variables.

Reply:
got it, warming up on my previous understanding of the code